src/Umbraco.Cms.Integrations.DAM.Aprimo/App_Plugins/UmbracoCms.Integrations/DAM/Aprimo/Render/AprimoAsset.cshtml

```diff
@@ -1,11 +1,53 @@
 ﻿@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<Umbraco.Cms.Integrations.DAM.Aprimo.Models.ViewModels.AprimoAssetViewModel>
 
-<link href="~/App_Plugins/UmbracoCms.Integrations/DAM/Aprimo/css/aprimo.css" rel="stylesheet"/>
+<style type="text/css">
+    .aprimo-asset-row {
+        display: flex;
+        justify-content: center;
+    }
+
+        .aprimo-asset-row .thumbnail {
+            float: left;
+            margin-right: 10px;
+        }
+
+        .aprimo-asset-row .text {
+            float: left;
+        }
+
+    .aprimo-crops-row {display: flex; width: 100%;}
+        .aprimo-crops-row .column {
+            flex: 50%;
+            padding: 10px;
+        }
+</style>
 
-<div class="aprimo-container">
+<div class="aprimo-asset-row">
     @if (!string.IsNullOrEmpty(Model.Thumbnail))
     {
-        <img class="aprimo-thumbnail" src="@Model.Thumbnail" alt="@Model.Title" />
+        <img class="thumbnail" src="@Model.Thumbnail" alt="@Model.Title" />
     }
-    <h3 class="aprimo-title">@Model.Title</h3>
-</div>
\ No newline at end of file
+    <div class="text">
+        <h3>@Model.Title</h3>
+    </div>
+</div>
+@if (Model.CropItemsVM.Any())
+{
+    <div class="aprimo-asset-row">
+        <h3>Crop Items</h3>
+        <br/>
+        <div class="aprimo-crops-row">
+            @foreach (var cropItem in Model.CropItemsVM)
+            {
+                <div class="column">
+                    @if (!string.IsNullOrEmpty(cropItem.PublicLink))
+                    {
+                        <img src="@cropItem.PublicLink" class="thumbnail" />
```

Comment:
Probably should render the `width` and `height` for the `img` tag using the `ResizeWidth` and `ResizeHeight` properties.

Similarly `Label` could be used for the `alt` attribute.